tree_structs.py

```diff
@@ -103,7 +103,7 @@ def scrub(s):
 def corenlp_to_xmltree_sub(s, dep_parents, rid=0):
```

Comment:
Also why isn't this file somewhere else?

Reply:
🤷‍♂️